libherokubuildpack/src/buildpack_output/mod.rs

```diff
@@ -560,6 +651,38 @@ mod test {
     use libcnb_test::assert_contains;
     use std::fs::File;
 
+    #[test]
+    fn background_timer() {
+        let io = BuildpackOutput::new(Vec::new())
+            .start_silent()
+            .section("Background")
+            .start_timer("Installing")
+            .finish()
+            .finish()
+            .finish();
+
+        // Test human readable timer output
+        let expected = formatdoc! {"
+            - Background
+              - Installing ... (< 0.1s)
+            - Done (finished in < 0.1s)
+        "};
+
+        assert_eq!(
+            expected,
+            strip_ansi_escape_sequences(String::from_utf8_lossy(&io))
+        );
+
+        // Test timer dot colorization
+        let expected = formatdoc! {"
+            - Background
+              - Installing\u{1b}[2;1m .\u{1b}[0m\u{1b}[2;1m.\u{1b}[0m\u{1b}[2;1m. \u{1b}[0m(< 0.1s)
+            - Done (finished in < 0.1s)
+        "};
+
+        assert_eq!(expected, String::from_utf8_lossy(&io));
+    }
+
```

Comment:
Intentionally not adding a changelog entry as the buildpack output is not (yet) shipped.

Reply:
Now that the buildpack output has shipped, a changelog entry is warranted here.

Reply:
I added a changelog